packages/models/src/bitcoin-unit.model.ts

```diff
@@ -0,0 +1,26 @@
+export enum WalletBitcoinUnit {
+  bitcoin = 'bitcoin',
+  satoshi = 'satoshi',
+}
+export type BitcoinUnit = keyof typeof WalletBitcoinUnit;
+
+export type BitcoinUnitSymbol = 'BTC' | 'sat';
+
+export interface BitcoinUnitInfo {
+  name: WalletBitcoinUnit;
+  symbol: BitcoinUnitSymbol;
+  decimal: string;
+}
+
+export const bitcoinUnitsKeyedByName: Record<WalletBitcoinUnit, BitcoinUnitInfo> = {
```

Comment:
We use a similar pattern for `networks`, should both be moved to `constants`?

Reply:
We have `networkModes` in `network.models`, maybe we can use those? 

I'm not sure I understand totally. I am happy to keep these types local to the model and load from there if needed rather than move to constants

Reply:
The idea of models is that it represents the structure of the data we use, not the data itself. Have also thought that network should be moved elsewhere, probably constants